src/gui/UBKeyboardPalette.cpp

```diff
@@ -274,8 +274,8 @@ void  UBKeyboardPalette::paintEvent( QPaintEvent* event)
 
     QRect r = this->geometry();
 
-    int lleft, ltop, lright, lbottom;
-    getContentsMargins ( &lleft, &ltop, &lright, &lbottom ) ;
+    const auto margins = contentsMargins();
+    int lleft, ltop = (margins.left(), margins.top());
```

Comment:
I'm not sure what this line actually does on C/C++. But anyway, it would be sufficient to declare and initialize these two variables later in lines 284 and 285.

Reply:
Oh right! The initial value is not even used.